profile_bluesky/startup/instrument/devices/ruby_motors.py

```diff
@@ -0,0 +1,21 @@
+
+__all__ = [
+    'ruby_mot'
+    ]
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from ophyd import Component, MotorBundle, EpicsMotor
+from ..framework import sd
+
+class RubyMotors(MotorBundle):
+  focus = Component(EpicsMotor,'m37',labels=('motor','ruby'))
```

Comment:
indentation of 4 spaces  is customary, blank space after `,`

Reply:
I created this file directly in GitHub, that's why the format is funny. Just fixed it.